### ISSUE TITLE
Build failed railway

### ISSUE THREAD

**Issue by makara4code** [kadumedim/directus-starter#41]
```
==============
Using Nixpacks
==============

context: 01257bbff961405ff290c85072bf8dfc

╔═════════════════════ Nixpacks v1.13.0 ════════════════════╗
║ setup      │ nodejs-18_x, npm-9_x                         ║
║───────────────────────────────────────────────────────────║
║ install    │ npm i                                        ║
║───────────────────────────────────────────────────────────║
║ start      │ npx directus bootstrap && npx directus start ║
╚═══════════════════════════════════════════════════════════╝


#0 building with "default" instance using docker driver

#1 [internal] load .dockerignore

#1 transferring context: 2B done
#1 DONE 0.0s

#2 [internal] load build definition from Dockerfile
#2 transferring dockerfile: 2.29kB done
#2 DONE 0.0s

#3 [internal] load metadata for ghcr.io/railwayapp/nixpacks:ubuntu-1690243402

#3 DONE 0.4s


#4 [stage-0 1/8] FROM ghcr.io/railwayapp/nixpacks:ubuntu-1690243402@sha256:1a7dc562bf1f13299c20e55f8a72a287cb16dfea072fe4d20d084cfe23494afd

#4 DONE 0.0s

#5 [internal] load build context
#5 transferring context: 6.46kB done
#5 DONE 0.0s

#6 [stage-0 2/8] WORKDIR /app/
#6 CACHED

#7 [stage-0 3/8] COPY .nixpacks/nixpkgs-293a28df6d7ff3dec1e61e37cc4ee6e6c0fb0847.nix .nixpacks/nixpkgs-293a28df6d7ff3dec1e61e37cc4ee6e6c0fb0847.nix
#7 CACHED

#8 [stage-0 4/8] RUN nix-env -if .nixpacks/nixpkgs-293a28df6d7ff3dec1e61e37cc4ee6e6c0fb0847.nix && nix-collect-garbage -d
#8 CACHED

#9 [stage-0 5/8] COPY . /app/.
#9 DONE 0.0s

#10 [stage-0 6/8] RUN --mount=type=cache,id=s/06233d0b-8b82-462a-a34d-210ecaa3e55a-/root/npm,target=/root/.npm npm i

#10 0.851 npm
WARN config production Use `--omit=dev` instead.

#10 17.82 npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility

#10 18.66 npm WARN deprecated @types/cssnano@5.1.0: This is a stub types definition. cssnano provides its own type definitions, so you do not need this installed.

#10 18.81 npm WARN deprecated har-validator@5.1.5: this library is no longer supported

#10 19.46 npm WARN deprecated consolidate@0.15.1: Please upgrade to consolidate v1.0.0+ as it has been modernized with several long-awaited fixes implemented. Maintenance is supported by Forward Email at https://forwardemail.net ; follow/watch https://github.com/ladjs/consolidate for updates and release changelog

#10 19.58 npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs

#10 20.53 npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

#10 23.87 npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

#10 24.03 npm WARN deprecated json2csv@5.0.7: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.

#10 41.13 npm ERR! code 1
#10 41.14 npm ERR! path /app/node_modules/isolated-vm
#10 41.14 npm ERR! command failed
#10 41.14 npm ERR! command sh -c -- prebuild-install || (node-gyp rebuild --release -j 4 && node-gyp clean)
#10 41.14 npm ERR! prebuild-install warn install /nix/store/wprxx5zkkk13hpj6k1v6qadjylh3vq9m-gcc-11.3.0-lib/lib/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /app/node_modules/isolated-vm/out/isolated_vm.node)
#10 41.14 npm ERR! gyp info it worked if it ends with ok
#10 41.14 npm ERR! gyp info using node-gyp@8.4.1
#10 41.14 npm ERR! gyp info using node@18.12.1 | linux | x64
#10 41.14 npm ERR! gyp ERR! find Python
#10 41.14 npm ERR! gyp ERR! find Python Python is not set from command line or npm configuration
#10 41.14 npm ERR! gyp ERR! find Python Python is not set from environment variable PYTHON

#10 41.14 npm ERR! gyp ERR! find Python checking if "python3" can be used
#10 41.14 npm ERR! gyp ERR! find Python - "python3" is not in PATH or produced an error
#10 41.14 npm ERR! gyp ERR! find Python checking if "python" can be used
#10 41.14 npm ERR! gyp ERR! find Python - "python" is not in PATH or produced an error
#10 41.14 npm ERR! gyp ERR! find Python
#10 41.14 npm ERR! gyp ERR! find Python **********************************************************
#10 41.14 npm ERR! gyp ERR! find Python You need to install the latest version of Python.
#10 41.14 npm ERR! gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
#10 41.14 npm ERR! gyp ERR! find Python you can try one of the following options:
#10 41.14 npm ERR! gyp ERR! find Python - Use the switch --python="/path/to/pythonexecutable"
#10 41.14 npm ERR! gyp ERR! find Python   (accepted by both node-gyp and npm)
#10 41.14 npm ERR! gyp ERR! find Python - Set the environment variable PYTHON
#10 41.14 npm ERR! gyp ERR! find Python - Set the npm configuration variable python:
#10 41.14 npm ERR! gyp ERR! find Python   npm config set python "/path/to/pythonexecutable"
#10 41.14 npm ERR! gyp ERR! find Python For more information consult the documentation at:
#10 41.14 npm ERR! gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
#10 41.14 npm ERR! gyp ERR! find Python **********************************************************
#10 41.14 npm ERR! gyp ERR! find Python
#10 41.14 npm ERR! gyp ERR! configure error
#10 41.14 npm ERR! gyp ERR! stack Error: Could not find any Python installation to use
#10 41.14 npm ERR! gyp ERR! stack     at PythonFinder.fail (/app/node_modules/node-gyp/lib/find-python.js:330:47)
#10 41.14 npm ERR! gyp ERR! stack     at PythonFinder.runChecks (/app/node_modules/node-gyp/lib/find-python.js:159:21)
#10 41.14 npm ERR! gyp ERR! stack     at PythonFinder.<anonymous> (/app/node_modules/node-gyp/lib/find-python.js:202:16)
#10 41.14 npm ERR! gyp ERR! stack     at PythonFinder.execFileCallback (/app/node_modules/node-gyp/lib/find-python.js:294:16)
#10 41.14 npm ERR! gyp ERR! stack     at exithandler (node:child_process:420:5)
#10 41.14 npm ERR! gyp ERR! stack     at ChildProcess.errorhandler (node:child_process:432:5)
#10 41.14 npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
#10 41.14 npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:289:12)
#10 41.14 npm ERR! gyp ERR! stack     at onErrorNT (node:internal/child_process:476:16)
#10 41.14 npm ERR! gyp ERR! stack     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
#10 41.14 npm ERR! gyp ERR! System Linux 5.10.0-24-cloud-amd64
#10 41.14 npm ERR! gyp ERR! command "/nix/store/7ppnyqmp1c92gkfrixbz99sw7q6ifkg1-nodejs-18.12.1/bin/node" "/app/node_modules/.bin/node-gyp" "rebuild" "--release" "-j" "4"
#10 41.14 npm ERR! gyp ERR! cwd /app/node_modules/isolated-vm
#10 41.14 npm ERR! gyp ERR! node -v v18.12.1
#10 41.14 npm ERR! gyp ERR! node-gyp -v v8.4.1
#10 41.14 npm ERR! gyp ERR! not ok
#10 41.14
#10 41.14 npm ERR! A complete log of this run can be found in:
#10 41.14 npm ERR!     /root/.npm/_logs/2023-08-28T07_14_27_272Z-debug-0.log
#10 ERROR: process "/bin/bash -ol pipefail -c npm i" did not complete successfully: exit code: 1

-----
> [stage-0 6/8] RUN --mount=type=cache,id=s/06233d0b-8b82-462a-a34d-210ecaa3e55a-/root/npm,target=/root/.npm npm i:
41.14 npm ERR! gyp ERR! stack     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
41.14 npm ERR! gyp ERR! System Linux 5.10.0-24-cloud-amd64
41.14 npm ERR! gyp ERR! command "/nix/store/7ppnyqmp1c92gkfrixbz99sw7q6ifkg1-nodejs-18.12.1/bin/node" "/app/node_modules/.bin/node-gyp" "rebuild" "--release" "-j" "4"
41.14 npm ERR! gyp ERR! cwd /app/node_modules/isolated-vm
41.14 npm ERR! gyp ERR! node -v v18.12.1
41.14 npm ERR! gyp ERR! node-gyp -v v8.4.1
41.14 npm ERR! gyp ERR! not ok
41.14
41.14 npm ERR! A complete log of this run can be found in:
41.14 npm ERR!     /root/.npm/_logs/2023-08-28T07_14_27_272Z-debug-0.log
-----

Dockerfile:20
-------------------
18 |     ENV NIXPACKS_PATH /app/node_modules/.bin:$NIXPACKS_PATH
19 |     COPY . /app/.
20 | >>> RUN --mount=type=cache,id=s/06233d0b-8b82-462a-a34d-210ecaa3e55a-/root/npm,target=/root/.npm npm i
21 |
22 |     # build phase
-------------------
ERROR: failed to solve: process "/bin/bash -ol pipefail -c npm i" did not complete successfully: exit code: 1

Error: Docker build failed

```

**Comment by kadumedim** [comment]
On it!

**Comment by kadumedim** [comment]
Just fixed it on the commit b6d3f52
If you have any problems please open a new Issue ;)
